platform/bazel/package.bzl

```diff
@@ -1,5 +1,5 @@
 load("@bazel_tools//tools/build_defs/pkg:pkg.bzl", "pkg_tar", "pkg_deb")
-load("//bazel:version.txt", "hash_compute")
+load("//bazel:version.bzl", "hash_compute", "version_compute")
```

Comment:
The version.txt -> version.bzl change should be in the hash_compute commit, if I understand correctly.